### ISSUE TITLE
Set up CI

### ISSUE THREAD

**Issue by carrascomj** [ztlpn/minilp#3]
Hi! 

I noticed there is not Continuous Integration for this one. Let me know if a PR (github actions I suppose) for that would be interesting.

**Comment by ztlpn** [comment]
Yes, that would be a welcome addition!

**Comment by carrascomj** [comment]
Cool! I'll submit one workflow with the starter pack **clippy, fmt, build and test**. 

I also saw that you have the same README as in the docs, so it could be automatically generated with [cargo-readme](https://github.com/livioribeiro/cargo-readme) like in [this one](https://github.com/carrascomj/rust_sbml/blob/trunk/.github/workflows/readme.yml) (the examples in github's would be always valid), but let's just go with CI for now.

**Comment by ztlpn** [comment]
Yeah, I've considered using cargo-readme, but ultimately decided against it because I wanted the README and the frontpage of the docs be a bit different.

**Comment by carrascomj** [comment]
Right, it's not always the best fit for every project. It also may be an overkill for only one example in the README.

Still, at least for now, it seems like the _README.md_ here just have additions with respect to the one from `lib.rs`, so something like this [_README.tlp_](https://github.com/carrascomj/rust_sbml/blob/trunk/README.tpl) could be used in this case (it is used by `cargo-readme` by defautl). If you are planning to make them more different it won't work, of course.
